extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -171,44 +222,54 @@ Backends need to be specified on the necessary components (binary-poll, poll, qu
 
 The interactive experience elements from `amp-story-interactive` share an API language for customizing options.
 
-### id (required for binary-poll, poll, quiz)
+### id (required for binary-poll, img-poll, img-quiz, poll, quiz)
 
 Element ID that identifies the interactive component in the story. Used to compose the [`interactiveId`](#interactiveId) sent to the backend. Should be unique for each component in the story.
 
-### endpoint (required for binary-poll, poll, quiz)
+### endpoint (required for binary-poll, img-poll, img-quiz, poll, quiz)
 
 Complete URL of backend. Stores interactive element votes.
 
 ### theme (optional)
 
 Controls the color of the chips and text. Can be `light` (default), `dark`.
 
-### chip-style (optional for poll, quiz, results)
+### chip-style (optional for img-poll, img-quiz, poll, quiz, results)
 
 Controls the style of the component. Can be `flat` (default), `shadow`, or `transparent`.
-Results and binary-poll elements don't support shadow.
+Results, binary-poll, img-poll, and img-quiz elements don't support shadow.
 
 ### prompt-text (optional)
 
 Adds a prompt to the top of the component. Use `prompt-text` to write the poll/quiz question, or as a prefix to the category in the `amp-story-interactive-result` element.
 
-### prompt-size (optional for binary-poll, poll, quiz)
+### prompt-size (optional for binary-poll, img-poll, img-quiz, poll, quiz)
 
 Controls the `font-size` of prompt text. Can be `small`, `medium` (default), `large`. Large prompts will hold up to 3 lines of text, other sizes will hold up to 4 lines of text.
 
 This attribute does not apply styling to `amp-story-interactive-result` category prefix text.
 
-### option-{1/2/3/4}-text (required)
+### option-{1/2/3/4}-text (required for binary-poll, poll, quiz, results)
 
 String that represents a numbered option. Binary polls require 2 options. Polls and quizzes may include between 2 and 4 options.
 
 The `amp-story-interactive-result` element uses this string value as category description.
 
-### option-{1/2/3/4}-confetti (optional for binary-poll, poll, quiz)
+### option-{1/2/3/4}-image (required for img-poll, img-quiz, optional for results)
+
+Path to an image that represents a numbered option, for anywhere between 2 to 4 options.
```

Comment:
Please add a suggested image dimension. 
This could be the ideal size for `amp-story-interactive-result` since the image would be displayed there if correct and need to look sharp.